manifests/c/Codeblocks/Codeblocks/20.03/Codeblocks.Codeblocks.installer.yaml

```diff
@@ -7,7 +7,9 @@ MinimumOSVersion: 10.0.0.0
 InstallerType: nullsoft
 Installers:
 - Architecture: x64
+  Scope: machine
```

Comment:
```suggestion
```

The installation is machine-wide as it installs to `C:\Program Files`, but the installer writes its uninstall entries to `HKEY_CURRENT_USER` which will make WinGet think it's a user scoped installation in the upgrade scenario and subsequent upgrades of this package will error out with `No applicable upgrade found` if we add `Scope: machine` in the manifest.

This is a similar scenario to https://github.com/microsoft/winget-pkgs/issues/94840 and the proper way to resolve this would be to have the publishers fix the installer to write the uninstall entries in the correct place.